CommandLine/emake/Makefile

```diff
@@ -16,18 +15,23 @@ else
 	OS_LIBS=-lboost_system-mt -Wl,--no-as-needed -Wl,-rpath,./ -lboost_program_options-mt -lboost_iostreams-mt -lboost_filesystem-mt -lboost_system-mt -lpthread
 endif
 
-PROTO_DIR := $(SHARED_SRC_DIR)/protos
-CXXFLAGS  := -I$(SRC_DIR) -I../../CompilerSource -I$(SHARED_SRC_DIR) -I$(PROTO_DIR)/.eobjs -std=c++11 -Wall -Wextra -Wpedantic -g
-LDFLAGS   := $(OS_LIBS) -L../../ -lProtocols -lprotobuf
+CXXFLAGS  += -I../../CompilerSource -I$(PROTO_DIR)
+LDFLAGS   += $(OS_LIBS) -L../../ -lProtocols -lprotobuf
 
-rwildcard=$(wildcard $1/$2) $(foreach d,$(wildcard $1/*),$(call rwildcard,$d,$2))
-SOURCES := $(call rwildcard,$(SRC_DIR),*.cpp)
+ifeq ($(TESTS), TRUE)
+	TARGET=../../emake-tests
+	SOURCES := $(call rwildcard, ../libEGM-test,*.cpp)
+	LDFLAGS += -lgtest -lpthread -lgtest_main
+else
+	TARGET = ../../emake
+  SOURCES := $(call rwildcard,$(SRC_DIR),*.cpp)
```

Comment:
This line starts with two spaces instead of a tab. It's curious that make is tolerating this.

Reply:
ill fix this just in case